src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -77,55 +68,62 @@ import org.sonarlint.intellij.trigger.TriggerType
+import org.sonarsource.sonarlint.core.rpc.protocol.client.connection.AssistCreatingConnectionParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.connection.AssistCreatingConnectionResponse
+import org.sonarsource.sonarlint.core.rpc.protocol.client.connection.GetCredentialsParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.connection.GetCredentialsResponse
+import org.sonarsource.sonarlint.core.rpc.protocol.client.event.DidReceiveServerHotspotEvent
+import org.sonarsource.sonarlint.core.rpc.protocol.client.event.DidReceiveServerTaintVulnerabilityChangedOrClosedEvent
+import org.sonarsource.sonarlint.core.rpc.protocol.client.event.DidReceiveServerTaintVulnerabilityRaisedEvent
+import org.sonarsource.sonarlint.core.rpc.protocol.client.fs.FindFileByNamesInScopeParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.fs.FindFileByNamesInScopeResponse
+import org.sonarsource.sonarlint.core.rpc.protocol.client.fs.FoundFileDto
+import org.sonarsource.sonarlint.core.rpc.protocol.client.hotspot.ShowHotspotParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.http.CheckServerTrustedParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.http.CheckServerTrustedResponse
+import org.sonarsource.sonarlint.core.rpc.protocol.client.http.GetProxyPasswordAuthenticationParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.http.GetProxyPasswordAuthenticationResponse
+import org.sonarsource.sonarlint.core.rpc.protocol.client.http.ProxyDto
+import org.sonarsource.sonarlint.core.rpc.protocol.client.http.SelectProxiesParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.http.SelectProxiesResponse
+import org.sonarsource.sonarlint.core.rpc.protocol.client.info.GetClientInfoResponse
+import org.sonarsource.sonarlint.core.rpc.protocol.client.issue.ShowIssueParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.log.LogLevel
+import org.sonarsource.sonarlint.core.rpc.protocol.client.log.LogParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.message.MessageType
+import org.sonarsource.sonarlint.core.rpc.protocol.client.message.ShowMessageParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.message.ShowSoonUnsupportedMessageParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.progress.ReportProgressParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.progress.StartProgressParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.smartnotification.ShowSmartNotificationParams
+import org.sonarsource.sonarlint.core.rpc.protocol.client.sync.DidSynchronizeConfigurationScopeParams
+import org.sonarsource.sonarlint.core.rpc.protocol.common.FlowDto
+import org.sonarsource.sonarlint.core.rpc.protocol.common.TextRangeDto
+import org.sonarsource.sonarlint.core.rpc.protocol.common.TokenDto
+import org.sonarsource.sonarlint.core.rpc.protocol.common.UsernamePasswordDto
+import java.io.ByteArrayInputStream
```

Comment:
Your imports settings don't look correctly configured according to Sonar standards. Please follow https://github.com/SonarSource/sonar-developer-toolset and then fix import order.